docs/openstack-zaqar.md

```diff
@@ -1,3 +1,5 @@
+!!! banner "TECH PREVIEW"
```

Comment:
Banner isn't a builtin type, this would need to conform to https://squidfunk.github.io/mkdocs-material/reference/admonitions - you could use "genestack" as an admonishment type?

```suggestion
!!! genestack "TECH PREVIEW"
```